great-idea-website/css/index.css

```diff
@@ -3,64 +3,339 @@
    License: none (public domain)
 */
 
```

Comment:
Awesome job on this. I love the color scheme you chose. One suggestion I would make would be to make the cursive font a little bigger, but other than that, everything looks amazing. If you want to push yourself further, I would try to get the fixed nav background to go across the entire window. Keep up the great work.